lib/public/components/Filters/RunsFilter/AliceL3AndDipoleFilteringModel.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RemoteData } from '/js/src/index.js';
+import { getRemoteData } from '../../../utilities/fetch/getRemoteData.js';
+import { SelectionDropdownModel } from '../../common/selection/dropdown/SelectionDropdownModel.js';
+
+/**
+ * AliceL3AndDipoleFilteringModel
+ */
+export class AliceL3AndDipoleFilteringModel extends SelectionDropdownModel {
+    /**
+     * Constructor
+     */
+    constructor() {
+        super({ multiple: false });
+    }
+
+    // eslint-disable-next-line valid-jsdoc
+    /**
+     * @inheritDoc
+     */
+    _initialize() {
+        getRemoteData('/api/runs/aliceCurrentLevels').then(({ data }) => {
+            this.setAvailableOptions(data
+                .sort(({ l3Level: l3LevelA, dipoleLevel: dipoleLevelA }, { l3Level: l3LevelB, dipoleLevel: dipoleLevelB }) =>
+                    l3LevelA - l3LevelB || dipoleLevelA - dipoleLevelB)
+                .map(({ l3Level, dipoleLevel }) => ({
+                    value: `${l3Level}kA/${dipoleLevel}kA`,
+                    filteringParams: {
+                        aliceL3Current: {
+                            [l3Level < 0 ? '>' : '>=']: l3Level * 1000 - 500,
+                            [l3Level < 0 ? '<=' : '<']: l3Level * 1000 + 500,
+                        },
+                        aliceDipoleCurrent: {
+                            [dipoleLevel < 0 ? '>' : '>=']: dipoleLevel * 1000 - 500,
+                            [dipoleLevel < 0 ? '<=' : '<']: dipoleLevel * 1000 + 500,
+                        },
+                    },
+                })));
```

Comment:
What is all that? For me, you should simply have a dropdown with pre-defined options: 
![image](https://github.com/AliceO2Group/Bookkeeping/assets/31805063/b31159ff-b1c3-4ec9-8601-7acece56d2d8)
And on select you can split the value or whatever

Reply:
And then I would need to create the same objects of filtering parameters, but extract levels from string additionally